src/App.jsx

```diff
@@ -14,45 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const MODE_ALPHABETICALLY = 'alphabetically';
+const MODE_LENGTH = 'length';
+const MODE_RESET = 'reset';
+
+function getPreparedGoods(goods, mode, reverse) {
+  goods.sort((good1, good2) => {
+    switch (mode) {
+      case MODE_ALPHABETICALLY:
+        return good1.localeCompare(good2);
+
+      case MODE_LENGTH:
+        return good1.length - good2.length;
+
+      case MODE_RESET:
+        return good1;
+
+      default:
+        return 0;
```

Comment:
your default case covers the reset option. you don't need an additional case for it.